firefox-ios/firefox-ios-tests/Tests/XCUITests/FakespotTests.swift

```diff
@@ -183,7 +182,7 @@ class FakespotTests: IphoneOnlyTestCase {
             let searchBestBuy = app.webViews["contentView"].textFields["Search"]
             mozWaitForElementToExist(searchBestBuy)
             searchBestBuy.tap()
-            searchBestBuy.typeText("iphone")
```

Comment:
![Simulator Screenshot - iPhone 15 - 2024-03-15 at 17 27 40](https://github.com/mozilla-mobile/firefox-ios/assets/1740517/13a84746-8237-4dee-a7ed-370571b73914)
Such a query does not return the products at the top of the page.

Reply:
Another alternative would be add a scroll before looking for `Apple`.